index.js

```diff
@@ -221,15 +222,28 @@ async function checkHeartbeat() {
         const innosentData = await redis.getLatestInnosentSensorData(location.locationid)
         latestRadar = moment(innosentData.timestamp, 'x')
       }
+
+      const doorData = await redis.getLatestDoorSensorData(location.locationid)
+      latestDoor = moment(doorData.timestamp, 'x')
       const currentTime = moment()
       const radarDelay = currentTime.diff(latestRadar)
+      const doorDelay = currentTime.diff(latestDoor)
+
+      const doorHeartbeatExceeded = doorDelay > helpers.getEnvVar('DOOR_THRESHOLD_MILLIS')
+      const radarHeartbeatExceeded = radarDelay > helpers.getEnvVar('RADAR_THRESHOLD_MILLIS')
 
-      if (radarDelay > RADAR_THRESHOLD_MILLIS && !location.heartbeatSentAlerts) {
-        helpers.logSentry(`Radar Heartbeat threshold exceeded; sending alerts for ${location.locationid}`)
+      if ((doorHeartbeatExceeded || radarHeartbeatExceeded) && !location.heartbeatSentAlerts) {
+        helpers.logSentry(`System at ${location.locationid} is disconnected`)
```

Comment:
Do you need this log considering you have the more specific logs directly below?